test/github-issues/11440/issue-11440.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from "chai"
+import "reflect-metadata"
+
+import { DataSource } from "../../../src"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { Post } from "./entity/Post"
+
+describe("github issues > #11440", () => {
+    let dataSource: DataSource
+    before(async () => {
+        ;[dataSource] = await createTestingConnections({
+            entities: [Post],
+            enabledDrivers: ["postgres"], // since the issue is for postgres only
+        })
+    })
+    beforeEach(() => reloadTestingDatabases([dataSource]))
+    after(() => closeTestingConnections([dataSource]))
```

Comment:
Since you're using `createTestingConnections`, you'll still need to store each data source here and pass the list into `reloadTestingDatabases` and `closeTestingConnections`. Then you'll need to iterate over each datasource in each test.

```suggestion
    let dataSources: DataSource[]
    before(async () => {
        dataSources = await createTestingConnections({
            entities: [Post],
            enabledDrivers: ["postgres"], // since the issue is for postgres only
        })
    })
    beforeEach(() => reloadTestingDatabases(dataSources))
    after(() => closeTestingConnections(dataSources))
```

Alternatively, you can take a look at using `setupSingleTestingConnection`, some tests use that configuration.

Reply:
I have updated the codebase, does it look okay now? Thanks